lib/web_ui/test/text_editing_test.dart

```diff
@@ -60,7 +61,9 @@ void main() {
   internalBootstrapBrowserTest(() => testMain);
 }
 
-void testMain() {
+Future<void> testMain() async {
+  await initializeEngine();
```

Comment:
I added this in the PR description. FlutterViewEmbedder initialization is now explicit. Previously, an accidental call to `flutterViewEmbedder` getter would automatically initialize it, so calling it accidentally inside the service initialization phase would also initialize the UI, which is what we're trying to prevent. Once I made it explicit, I immediately found all the places in `initializeEngineServices` that I needed to fix.

Reply:
Ahhh, gotcha, didn't associate this change with the auto-initialization of the flutterViewEmbedder!